src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts

```diff
@@ -0,0 +1,72 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { escape } from 'vs/base/common/strings';
+import { localize } from 'vs/nls';
+
+export function used() {
+}
+
+export default () => `
+<div class="welcomePageContainer">
+	<div class="welcomePage">
+		<div class="title">
+			<h1 class="caption">${escape(localize('welcomePage.azdata', "Azure Data Studio"))}</h1>
+			<p class="subtitle detail">${escape(localize({ key: 'welcomePage.azdataTagline', comment: ['Shown as subtitle on the Welcome page.'] }, ""))}</p>
+		</div>
+		<div class="row">
+			<div class="splash">
+				<div class="section start">
+					<h2 class="caption">${escape(localize('welcomePage.start', "Start"))}</h2>
+					<ul>
+						<li><a href="command:registeredServers.addConnection">${escape(localize('welcomePage.newConnection', "New connection"))}</a></li>
+						<li><a href="command:workbench.action.files.newUntitledFile">${escape(localize('welcomePage.newQuery', "New query"))}</a></li>
+						<li><a href="command:notebook.command.new">${escape(localize('welcomePage.newNotebook', "New notebook"))}</a></li>
+						<li class="mac-only"><a href="command:workbench.action.files.openLocalFileFolder">${escape(localize('welcomePage.openFileMac', "Open file"))}</a></li>
+						<li class="windows-only linux-only"><a href="command:workbench.action.files.openFile">${escape(localize('welcomePage.openFileLinuxPC', "Open file"))}</a></li>
```

Comment:
dup?

Reply:
Are we going to add Open Folder for next release?

Reply:
I fixed the 2 links issue; will only see 1.

I worked with @yualan who came up with the list of items to put on the welcome page. Let's ask him 😄